easybuild/easyconfigs/e/EggLib/EggLib-2.1.10-intel-2015b.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonPackage'
+
+name = 'EggLib'
+version = '2.1.10'
+
+homepage = 'http://egglib.sourceforge.net/'
+description = """EggLib is a C++/Python library and program package for evolutionary genetics and genomics."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [
+    '%(namelower)s-cpp-%(version)s.tar.gz',
+    '%(namelower)s-py-%(version)s.tar.gz',
+]
+
+dependencies = [
+    ('Python', '2.7.11'),
```

Comment:
We'll have to move to `.11` sooner or later. I already did for DOLFIN, for example.

And we're really close to moving on to `intel/2016a` now, so jumping to Python 2.7.11 is going to happen anyway.